snowpack/src/build/optimize.ts

```diff
@@ -420,8 +401,9 @@ async function runEsbuildOnBuildDirectory(
         addTrailingSlash(config.buildOptions.out),
       )),
   );
+  const manifest = metafile!;
   if (!config.optimize?.bundle) {
-    delete manifest.outputs;
+    manifest.outputs = {};
```

Comment:
lets keep this as deleted for now. Let me know if you need help making this work with TS. I assume that we can override the type in the `const manifest =` asignment, something like:

```
const manifest: SnowpackOptimizeManifest = metafile!;
if (!config.optimize?.bundle) {
    delete manifest.outputs;
```

Reply:
I have tried to express that the manifest is an `esbuild.Manifest` until we remove the `outputs` keys, let me know if you prefer other types.